### PR TITLE
#160520896 ft-get-all-orders 

### PR DESCRIPTION
**Pull request by Johnpaul-Borree**
#### What does this PR do?
- Enable users to view orders history
#### Description of Task to be completed?
- create controller actions to get all orders history
- create a route for this endpoint
- display messages on json string
#### How should this be manually tested?
- Test API endpoint on postman using Get /api/v1/orders
- but before you do so, npm run start
#### What are the relevant pivotal tracker stories?
#160520896

### CODE REVIEW

**Review by KaiserPhemi — COMMENTED** [review]
#### Nitpick
Please fix the title of the PR (and all PRs going forward) to follow best practices/convention

File: src/controller/ordersControllers.js [hunk 1]
```diff
@@ -1,37 +1,40 @@
...
+		type: Joi.string().min(3).required(),
+		orderDate: Joi.string().min(6).required(),
+		expires: Joi.string().min(6).required(),
+		quantities: Joi.number().integer().min(1).max(50000).required(),
+	};
 
-    return Joi.validate(order, schema);
+	return Joi.validate(order, schema);
     
 }
```
Comment (KaiserPhemi) [inline src/controller/ordersControllers.js]: #### Nitpick
Add a new line

File: src/controller/ordersControllers.js [hunk 2]
```diff
@@ -1,37 +1,40 @@
...
+	};
 
-    orders.push(order);
+	orders.push(order);
 
-    res.json(order);
+	res.json({justAdded: order, message: 'order Created'});
 }
 
 function validateOrders(order) {
```
Comment (KaiserPhemi) [inline src/controller/ordersControllers.js]: #### Critical
Same feedback on #11

**Review by KaiserPhemi — CHANGES_REQUESTED** [review]
Attend to feedback

File: src/controller/ordersControllers.js (hunk 1 above)
Comment (Johnpaul-Borree) [reply src/controller/ordersControllers.js]: Thanks, done

File: src/controller/ordersControllers.js (hunk 2 above)
Comment (Johnpaul-Borree) [reply src/controller/ordersControllers.js]: Thanks so much, it has been taken care of

**Pull request merged**